libraries/AP_HAL_Linux/RCInput_RPI.cpp

```diff
@@ -167,6 +167,7 @@ Memory_table::Memory_table(uint32_t page_count, int version)
 
     if ((fdMem = open("/dev/mem", O_RDWR | O_SYNC | O_CLOEXEC)) < 0) {
         fprintf(stderr, "Failed to open /dev/mem\n");
+        printf("Make sure that CONFIG_STRICT_DEVMEM is disabled\n");
```

Comment:
I added this line here as well. as it is called first and app will exit if it fails.